src/app/user.service.ts

```diff
@@ -11,7 +11,7 @@ export class UserService {
   constructor(private http: HttpClient) { }
 
   getCurrentUser() {
-    return this.http.get<User>('/api/users/currentUser')
+    return this.http.get<User>('/api/user/current')
```

Comment:
Route should be `/api/users/current` per resource oriented api. Please correct this here and on back-end if necessary. @noxyb0x